olp-cpp-sdk-core/src/cache/DefaultCacheImpl.cpp

```diff
@@ -828,6 +828,10 @@ DefaultCacheImpl::GetFromDiscCache(const std::string& key) {
   return boost::none;
 }
 
+uint64_t DefaultCacheImpl::GetProtectedCacheSize() const {
+  return protected_cache_->Size();
```

Comment:
Please check that `protected_cache_` is not `nullptr` before. There is not always a protected cache set.